rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -63,12 +64,12 @@ def __next__(self):
             self.bboxes_label_count = np.zeros(self.bs, dtype="int32")
             self.loader.rocalGetBoundingBoxCount(self.bboxes_label_count)
             # 1D labels array in a batch
-            self.labels = self.loader.rocalGetBoundingBoxLabels()
+            self.labels = self.loader.get_bounding_box_labels()
             # 1D bboxes array in a batch
-            self.bboxes = self.loader.rocalGetBoundingBoxCords()
-            #Image sizes of a batch
-            self.img_size = np.zeros((self.bs * 2),dtype = "int32")
-            self.loader.GetImgSizes(self.img_size)
+            self.bboxes = self.loader.get_bounding_box_cords()
+            # Image sizes of a batch
+            self.img_size = np.zeros((self.batch_size * 2), dtype="int32")
```

Comment:
Why are we multiplying by 2?

Reply:
It returns img sizes as pair of (height,width) so it'll require a total of BS*2 ints to copy the image sizes